src/test/java/calculator/InputCheckingTest.java

```diff
@@ -0,0 +1,29 @@
+package calculator;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.ValueSource;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+class InputCheckingTest {
+
+	InputChecking inputCheckingTest = new InputChecking();
+
+	@DisplayName("인자의 값이 빈칸 또는 null 인지 점검한다.")
+	@Test
+	void shouldCheckIsTheInputBlankTest() {
+		String isBlank = "";
+		String isNull = null;
+		assertThat(inputCheckingTest.isBlank(isBlank)).isEqualTo(true);
+		assertThat(inputCheckingTest.isBlank(isNull)).isEqualTo(true);
```

Comment:
1. `InputCheckingTest` 의 `isBlank` 메서드는 static 메서드인데 인스턴스를 생성해서 인스턴스에서 호출해서 사용하고 있네요!

2. 항상 참인 단언문은 아래와 같이 바꿀 수 있습니다!

```suggestion
		assertThat(inputCheckingTest.isBlank(isBlank)).isTrue();
		assertThat(inputCheckingTest.isBlank(isNull)).isTrue();
```

Reply:
넵 반영했습니다! 감사합니다.